2_process/src/process_river_geoms.R

```diff
@@ -1,16 +1,15 @@
 process_river_geoms <- function(ind_file,
                                 major_river_geoms_ind,
-                                gage_river_geoms_ind,
```

Comment:
We don't want to commit any changes related to skipped `gage_river_geoms` - these were special for getting Henri out the door. We want to keep the `main` branch clean.